duckduckgo/TrackerRadarKit
# Set up GitHub Actions

Task/Issue URL: https://app.asana.com/0/1200194497630846/1202595601161164/f
Tech Design URL:
CC: @ayoy 

**Description**:

This PR migrates TrackerRadarKit to GitHub Actions.

**Steps to test this PR**:
1. Push a deliberate test failure and check that it fails
1. Remove the previous change, then push a deliberate compilation failure and check that it fails
1. Push a SwiftLint violation and check that it fails. Note that this only fails on strict SwiftLint violations – should it fail on _all_ violations instead?



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
